Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Company/Company.tsx

```diff
@@ -31,18 +31,19 @@ import AdditionalFieldsWindow from '../CommonComponents/AdditionalFieldsWindow';
 import CompanyInfoWindow from './CompanyInfo';
 import CompanyMeterWindow from './CompanyMeter';
 import { LoadingScreen, Warning } from '@gpa-gemstone/react-interactive';
+import { OpenXDA } from '@gpa-gemstone/application-typings'
```

Comment:
I think you can remove `import { SystemCenter } from '../global';` since we moved most of this to the gpa-gemstone packages